contrib/inventory_builder/tests/test_inventory.py

```diff
@@ -209,8 +209,8 @@ def test_purge_invalid_hosts(self):
             ('doesnotbelong2', {'whateveropts=ilike'})])
         self.inv.yaml_config['all']['hosts'] = existing_hosts
         self.inv.purge_invalid_hosts(proper_hostnames)
-        self.assertTrue(
-            bad_host not in self.inv.yaml_config['all']['hosts'].keys())
+        self.assertNotIn(
+            bad_host, self.inv.yaml_config['all']['hosts'].keys())
```

Comment:
I think that this works without having to specify the keys() method
```suggestion
            bad_host, self.inv.yaml_config['all']['hosts'])
```